repository/auth.ts

```diff
@@ -104,4 +104,26 @@ const verify = async ({ token }: { token: string }): Promise<boolean> => {
     }
 };
 
-export { login, register, verify };
+const sendResetEmail = async (email: string) => {
+    try {
+        // Simulate an asynchronous operation that takes 5 seconds to complete
+        await new Promise((resolve) => setTimeout(resolve, 5000));
+        return { success: true };
+    } catch (error) {
+        return { success: false, error: { message: `Error sending email to ${email}` } };
+    }
```

Comment:
For now return only False, until we push the logic with real backend. So that user feels that it is really failing, not a problem with their id.